datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMap.java

```diff
@@ -157,6 +162,14 @@ private String getQueryString(Expression expression) {
       return null;
     }
 
+    //maximum records to return
+    if (strQuery.contains(",") && strQuery.indexOf(",") != strQuery.length() - 1) {
```

Comment:
What should be the correct format in the strQuery?
Please add description for it.